src/app/pages/vm/devices/device-add/device-add.component.ts

```diff
@@ -402,11 +404,16 @@ export class DeviceAddComponent implements OnInit, OnDestroy {
 
     // pci
     this.ws.call('vm.device.passthrough_device_choices').pipe(untilDestroyed(this)).subscribe((res) => {
+      this.isLoadingPci = true;
```

Comment:
Why set `isLoadingPci = true` after the call has already been subscribed to? At that point, you've already got the response. The loader won't show up while the call is loading but will show up for half a millisecond once we get the response. This line should be moved outside the subscribe on line 405 or something.